security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -201,6 +188,18 @@ func (s *sdsservice) StreamSecrets(stream sds.SecretDiscoveryService_StreamSecre
 			con.ResourceName = resourceName
 			con.mutex.Unlock()
 
+			conIDresourceNamePrefix := sdsLogPrefix(conID, resourceName)
+			if !s.skipToken && token == "" {
```

Comment:
remove checking token=="" here since some token change in each request.

Reply:
Thanks. I have removed this checking. Before this PR, we don't change token. Once a new connection comes, we get token and reuse it until connection closes. Now we fetch token for every request. Ideally we need a test to cover this change.